Ghidra/Processors/ARM/data/languages/ARMneon.sinc

```diff
@@ -4676,14 +4676,14 @@ vpushSd64List: "{"^buildVpushSd64List^"}"	is TMode=1 & thv_D22 & thv_c1215 & thv
 }
 
 buildVpopSdList: Sreg					is counter=0 & Sreg			[ regNum=regNum+1; ]
-   { tmp:4 = *mult_addr; Sreg = zext(tmp); mult_addr = mult_addr + 4; }
+   { tmp:4 = *mult_addr; Sreg = zext(tmp); mult_addr = mult_addr - 4; }
```

Comment:
The operational encoding for vldm uses addition, not subtraction. The sleigh for the vldm* instructions do the same thing. So why is this instruction that is an alias for vldm using subtraction?